lending/loan_management/utils.py

```diff
@@ -77,45 +77,47 @@ def get_payment_entries_for_bank_clearance(
 	return entries
 
 
-def get_matching_vouchers_for_bank_reconciliation(
+def get_matching_queries(
 	bank_account,
 	company,
 	transaction,
 	document_types,
+	exact_match,
+	account_from_to,
 	from_date,
 	to_date,
 	filter_by_reference_date,
 	from_reference_date,
 	to_reference_date,
-	exact_match,
-	filters,
 ):
-	vouchers = []
+	queries = []
 
 	if transaction.withdrawal > 0.0 and "loan_disbursement" in document_types:
-		vouchers.extend(get_ld_matching_query(bank_account, exact_match, filters))
+		queries.extend(get_ld_matching_query(bank_account, exact_match, transaction))
```

Comment:
Since `get_ld_matching_query` returns a string
```suggestion
		queries.append(get_ld_matching_query(bank_account, exact_match, transaction))
```